integration_tests/models/sql/test_unpivot_quote.sql

```diff
@@ -0,0 +1,26 @@
+{% if target.name == 'redshift' %}
+-- redshift lowercases all columns. ignoring this test for redshift
+select * from {{ ref('data_unpivot_quote_expected') }}
+
+{% else %}
+
+
+select
+    {{ adapter.quote("customer_id") }},
+    {{ adapter.quote("created_at") }},
+    {{ adapter.quote("prop") }},
+    {{ adapter.quote("val") }}
```

Comment:
Do these actually need to be quoted?

Reply:
BigQuery seems to have problems with unquoted columns since it uppercases all columns - I will doublecheck this to be sure

Reply:
Quoting the column names is required for snowflake. Other tests handle this conditionally:

https://github.com/dbt-labs/dbt-utils/blob/965726ea4a1cf2d793ec93ade5d39a884c8dc577/integration_tests/models/sql/test_unpivot_bool.sql#L8-L12

Reply:
@joellabes I added a comment mentioning the cause for the quotes in the test itself. Is this sufficient?